release-notes/10.0/preview/preview1/winforms.md

```diff
@@ -2,13 +2,106 @@
+
+For applications that still rely on `BinaryFormatter` for custom format serialization or need to maintain complete compatibility with .NET Framework applications, a reference to the out-of-band, unsupported `System.Runtime.Serialization.Formatters` package and the corresponding AppContext switch is required as detailed [here](https://learn.microsoft.com/en-us/dotnet/standard/serialization/binaryformatter-migration-guide/compatibility-package). This is particularly necessary for scenarios where complex types are involved, and the application cannot migrate to the new JSON-based APIs. In such cases, developers must explicitly opt-in to enable BinaryFormatter serialization and deserialization by setting the `Windows.ClipboardDragDrop.EnableUnsafeBinaryFormatterSerialization` switch to `true` in the configuration file in addition to `System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization` switch to `true`. The advantage to this approach is that `BinaryFormatter` can be explicitly disabled for Clipboard scenarios, while still being allowed in other parts of the application. This ensures that the application can continue to function while gradually transitioning to safer alternatives.
+
+For applications that still rely on `BinaryFormatter` for custom format serialization or need to maintain complete compatibility with .NET Framework applications, a reference to the out-of-band, unsupported `System.Runtime.Serialization.Formatters` package and the corresponding AppContext switch is required as detailed [here](https://learn.microsoft.com/en-us/dotnet/standard/serialization/binaryformatter-migration-guide/compatibility-package). This is particularly necessary for scenarios where complex types are involved, and the application cannot migrate to the new JSON-based APIs. In such cases, developers must explicitly opt-in to enable BinaryFormatter serialization and deserialization by setting the `Windows.ClipboardDragDrop.EnableUnsafeBinaryFormatterSerialization` switch to `true` in the configuration file in addition to `System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization` switch to `true`. The advantage to this approach is that `BinaryFormatter` can be explicitly disabled for Clipboard scenarios, while still being allowed in other parts of the application. This ensures that the application can continue to function while gradually transitioning to safer alternatives.
+
+1. *Replace obsolete APIs*  Start by replacing the obsolete `GetData` methods with the new `TryGetData<T>`. This ensures type safety and avoids the risks associated with unbounded deserialization.
+
+1. *Use JSON Serialization:* For complex types, use JSON serialization instead of BinaryFormatter, replace `SetData` with the new APIs like `SetDataAsJson`. The `TryGetData<T>` APIs are designed to handle common .NET types and JSON serialization.

```

Comment:
This looks like duplicated content

Reply:
It is a bit repetitive, let me tweak the summary paragraph to not be so detailed since the list covers it.